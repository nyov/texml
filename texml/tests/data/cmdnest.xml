<TeXML>
	<cmd name="par1">
		<opt>
			<cmd name="par2">
				<opt>
					<cmd name="par3">
						<parm>
							<cmd name="par4"/>
						</parm>
					</cmd>
				</opt>
			</cmd>
		</opt>
		<parm>aaa</parm>
	</cmd>
</TeXML>
