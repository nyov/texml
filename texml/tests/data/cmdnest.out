%
	\par1[
			\par2[
					\par3[&#x7C0;]{
							\par4 
						}
				]
		]{aaa}
